### ISSUE TITLE
[FEATURE] Needs ability to turn off showing up on new tabs

### ISSUE THREAD

**Issue by intrepidsilence** [yuanzhoucq/Tab-Space#42]
**Is your feature request related to a problem? Please describe.**
When opening a new tab, Tab Space inserts itself as the page it opens. This is not desired. This also prevents the cursor from being in the URL bar so one cannot start typing right after using the CMD+T shortcut as is the default in Safari. This breaks people's workflow.

**Describe the solution you'd like**
* Allow for the new tab start page to not open Tab Space
* If Tab Space is allowed to be the page opened with a new tab then ensure the cursor ends up in the URL bar, ready for typing.

**Describe alternatives you've considered**
The alternative right now is to turn off Tab Space and lose the $ spent on it as this makes Safari unbearable to use.

**Comment by yuanzhoucq** [comment]
Hello, you mean Tab Space was automatically set as the default page of Safari's new tab? How could it be possible 😂 Or do I misunderstand what you describe? Please correct me, thanks 😄

**Comment by intrepidsilence** [comment]
Yes that is exactly what I mean and I could not turn it off without disabling the plugin.  

--  
Michael Spurlock

**Comment by yuanzhoucq** [comment]
> Yes that is exactly what I mean and I could not turn it off without disabling the plugin.

Could you please share the "General" pane of your Safari's preferences window?

**Comment by intrepidsilence** [comment]
Well, it resolved itself. I went to take a photo of my General tab and it was set as expected on "Empty Page" for all new session types. So I went back and re-enabled Tab Space extension and opened a new tab and it was empty as it should be. I did recently upgrade to the latest beta of 11.1 Big Sur. I wonder if that fixed something. Anyway, thanks for being available and if it does it again I will let you know.
